src/foam/dao/EasyDAO.js

```diff
@@ -201,13 +201,13 @@ if ( getOrder() != null &&
   }
 }
 
-if ( getAuthorize() || getAuthorizer() != null ) {
-  if ( getAuthorizer() != null ) delegate = new foam.nanos.auth.AuthorizationDAO(getX(), delegate, getAuthorizer(), getAuthorizeReads());
+if ( getAuthorizer() != null ) delegate = new foam.nanos.auth.AuthorizationDAO(getX(), delegate, getAuthorizer());
+else if ( getAuthorize() ) {
```

Comment:
Should this be 'else'? Could I do both?

Reply:
if you want to use the default authorizers you can extend them in the custom authorizer